### ISSUE TITLE
I cant install cheat engine

### ISSUE THREAD

**Issue by Ayskos** [cheat-engine/cheat-engine#2202]
So first of all, when I run the installer the loading bar comes up and loads for ages, after a really long time the next screen comes up and i click next, but then it pops up saying there was an error installing cheat engine.

I looked at an issue, where it looked like he had the same problem as myself #2188 
I did everything that he was told to try but nothing seems to work for me, I also looked at #2099

After i click finish when the cheat engine install error pops up, it comes up with a runtime error picture attached under ↓
I would really appreciate if anyone could help me with this.

![Screenshot (5)](https://user-images.githubusercontent.com/106444710/170835388-6a1790d6-cc66-48a4-8863-5b500243f3fb.png)

**Comment by fade2gray** [comment]
Did you also check #1954? Where the video instructs to add an exclusion for the downloads folder, at about 1min 20 secs, also try adding an exclusion for your temp folder `C:\Users\<username>\AppData\Local\Temp`.

Also, take a look at [this](https://github.com/cheat-engine/cheat-engine/issues/2020#issuecomment-1031519053).

**Comment by Ayskos** [comment]
> Did you also check #1954? Where the video instructs to add an exclusion for the downloads folder, at about 1min 20 secs, also try adding an exclusion for your temp folder `C:\Users\<username>\AppData\Local\Temp`.
> 
> Also, take a look at [this](https://github.com/cheat-engine/cheat-engine/issues/2020#issuecomment-1031519053).

I've added my Temp folder as an exclusion and used the method from #1954 , but none of it worked.
I've also allowed the installer to go through my firewall, but that didnt work either, now I don't know what else to try.

I also looked at #2020 where the runtime error comes up for him too, and i personally use malwarebytes so it isn't a virus 
"not that i'm aware of" and i'm pretty sure the installer has access to the internet, not sure how to check that tho.

Not that it will help i don't know, but in my task manager the installer sometimes show as not responding, while the loading bar is on screen.

**Comment by fade2gray** [comment]
Are you disabling all the real-time protection layers in malwarebytes?

Maybe your ISP is blocking the installer from completing the download, Can you connect your PC using a mobile device as a hotspot - see [this](https://github.com/cheat-engine/cheat-engine/issues/2165#issuecomment-1133829108)

**Comment by Ayskos** [comment]
> Are you disabling all the real-time protection layers in malwarebytes?
> 
> Maybe your ISP is blocking the installer from completing the download, Can you connect your PC using a mobile device as a hotspot - see [this](https://github.com/cheat-engine/cheat-engine/issues/2165#issuecomment-1133829108)

Yes everything is turned off, and i've even put the folders and installer on the allow list just to make sure.

I've previously had cheat engine installed, and I don't recall having any problems then, but I had to factory reset my pc's main drive and now i'm trying to download cheat engine again but it just isn't working.

**Comment by fade2gray** [comment]
@Ayskos Glad you sorted it. Any chance you can name the 'program' or what the intended function is?
